docs/synapse/userguides/design_forms_vs_tags.rst

```diff
@@ -1,62 +1,180 @@
-
-Using a tag (such as applying ``#cno.threat.t12.use`` to the ``inet:ipv4`` node) gives us the most flexibility in recording the information that the IP was associated with a specific set of malicious activty. In addition, if we know "when" the IP was used by or associated with the threat cluster, we can leverage tag timestamps to record that information.
+We also recommend desiging forms broadly - this may require some out-of-the-box thinking to consider how the form may
+apply to other fields, disciplines, or even locales ("how something works" in the United States may be different from how
+it works in Argentina or Malaysia).
+
+.. _design-props:
+
+Properties
+----------
+
+Properties are details that further define a form. When creating a form, there are probably a number of "things you want
+to record" about the form that immediately come to mind. These are obvious candidates for properties.
+
+A few considerations when designing properties:
+
+- Properties should be highly "intrinsic" to their forms. The more closely related something is to an object, the more
+  likely it should be a property of that object. Things that are not highly intrinsic are better candidates for their own
+  forms, for "relationship" forms, or for tags.
+
+- Consider whether a property has enough "thinghood" to also be its own form (and possibly type).
+
+- The data model supports multi-value :ref:`type-array` properties, but arrays are not meant to store an excessive
+  number of values (largely for performance and visualization purposes). In this situation, a "relationship" form
+  might be preferable. Another option would be to "reverse" the property relationship.
+  
+  For example, a compromise (``risk:compromise``) may consist of a number of different attacks (``risk:attack`` nodes)
+  representing steps in the overall compromise. Instead of ``risk:compromise`` having an ``:attacks`` array with a
+  large number of values, a ``risk:attack`` has a ``:compromise`` property so that multiple attacks can be linked
+  back to a single compromise.
+
+.. _design-edges:
+
+Light Edges
+-----------
+
+While it is preferable to represent most relationships as forms (which can record additional properties as well as
+tags), light edges can replace "relationship" forms in cases where additional properties or tags are unnecessary.
+
+A more important criteria is where the object on one or both sides of the relationship could be any object.
```

Comment:
This sentence reads a little weird to me, but maybe i've just been reading too many docs.

Reply:
Rewrote this bit. I've been *writing* too many docs. :-)